data-access-services/pom.xml

```diff
@@ -49,6 +49,15 @@
                         </excludes>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.1.2</version>
+                    <configuration>
+                        <configLocation>../config/checkstyle/checkstyle.xml</configLocation>
+                        <violationSeverity>warning</violationSeverity>
```

Comment:
Should we check test sources too?

`<includeTestSourceDirectory>true</includeTestSourceDirectory>`

Reply:
sure, updated.